content/en/docs/setup/other_config/server-group-launch-settings/aws-ec2/_index.md

```diff
@@ -9,13 +9,14 @@ description: An AWS EC2 Server Group offers a number of launch setting configura
 An AWS Server Group can be set up with either a launch template or a launch configuration. However, <b>you must enable launch templates support for all of your applications if you want to use the latest AWS features.
 AWS strongly recommends using _launch templates_ over _launch configurations_ because launch configurations do NOT provide full functionality for Amazon EC2 Auto Scaling or Amazon EC2.</b>
 
-* (Newer) [launch template](https://docs.aws.amazon.com/autoscaling/ec2/userguide/LaunchTemplates.html) is similar to a launch configuration, in that it specifies instance configuration information.
+* (Newer) [Launch template](https://docs.aws.amazon.com/autoscaling/ec2/userguide/LaunchTemplates.html) is similar to a launch configuration, in that it specifies instance configuration information.
    However, defining a launch template instead of a launch configuration, you can:
    - create and maintain multiple versions of a launch template
-   - access the launch-template-only AWS features like diversification of instances of a server group across instance type, purchase options (On-Demand / Spot), allocation strategies. 
-   To enable launch templates support, follow guidelines in <b>[AWS EC2 Launch Templates Setup](/docs/setup/other_config/server-group-launch-settings/aws-ec2/launch-templates-setup)</b>
-   To learn more about launch templates features, see <b>[AWS EC2 Launch Templates](/docs/setup/other_config/server-group-launch-settings/aws-ec2/launch-templates)</b>
-* (Older) [launch configuration](https://docs.aws.amazon.com/autoscaling/ec2/userguide/LaunchConfiguration.html) is an instance configuration template that an AWS Auto Scaling group uses to launch EC2 instances.
+   - access the launch-template-only AWS features like diversification of instances of a server group across instance type, purchase options (On-Demand / Spot), allocation strategies.
+     * <b>[Enable launch templates support](/docs/setup/other_config/server-group-launch-settings/aws-ec2/launch-templates-setup)</b>
+     * <b>[Learn more about launch template features](/docs/setup/other_config/server-group-launch-settings/aws-ec2/launch-templates)</b>
```

Comment:
```suggestion
     * [Enable launch templates support](/docs/setup/other_config/server-group-launch-settings/aws-ec2/launch-templates-setup).
     * Learn more about launch template features](/docs/setup/other_config/server-group-launch-settings/aws-ec2/launch-templates).
```

Reply:
Links are already formatted differently from txt, so no real reason to bold them too.